task/github-request-reviewers/0.1/github-request-reviewers.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: github-request-reviewers
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.17.0"
+    tekton.dev/tags: github
+    tekton.dev/displayName: "request github reviewers"
+spec:
+  description: >-
+    This Task will request reviewers for a GitHub PR
+
+    It can take either a list of users or teams or both.
+    Note that, when teams are used, GitHub does not verify
+    if the team (or any of its members) are collaborators
+    the repo, so a 201 reply may still result in no reviewer
+    added to the PR.
+  workspaces:
+    - name: github
+      description: The workspace that contains the GitHub secret
+  params:
+    - name: GITHUB_HOST_URL
+      description: |
+        The GitHub host, adjust this if you run a GitHub Enteprise.
+      default: "api.github.com"
+      type: string
+
+    - name: API_PATH_PREFIX
+      description: |
+        The API path prefix, GitHub Enterprise has a prefix e.g. /api/v3
+      default: ""
+      type: string
+
+    - name: PACKAGE
+      description: "{owner}/{repo}"
+      type: string
+
+    - name: PULL_REQUEST_NUMBER
+      description: The pull request number
+      type: string
+
+    - name: REVIEWERS
+      description: |
+        Command separated list of github user slugs.
+      type: string
+      default: ""
+
+    - name: TEAM_REVIEWERS
+      description: |
+        Command separated list of github team slugs.
+      type: string
+      default: ""
+
+    - name: GITHUB_TOKEN_FILE
+      description: |
+        The file within workspace that contains the GitHub token.
+      type: string
+      default: token
+
+  steps:
+  - name: set-reviewers
+    image: python:3.9-alpine3.13
```

Comment:
consider using a fully qualified name - e.g. docker.io/library/python:3.9-alpine3.13@sha256:abcdef

Reply:
ok

Reply:
Uhm, I'm a bit confused by how the image is stored on dockerhub: 

https://hub.docker.com/layers/python/library/python/3.9-alpine3.13/images/sha256-d73b7b47b0d15fa47a638a6a271ae88abbe532168f5626156ac091dcd543001b?context=explore

It looks like each sha points to a specific architecture. 
I'm not sure if that means that by using a tag the image for the correct architecture will be pulled?

Trying it locally
```
$ docker pull python:3.9-alpine3.13
3.9-alpine3.13: Pulling from library/python
ba3557a56b15: Pull complete
996c4f0e31fa: Pull complete
476a615c7791: Pull complete
1f2ae9e45137: Pull complete
cdcfcfbaf716: Pull complete
Digest: sha256:437482dd314471f5f10967f7376489d9e7f67c111850c123cc725a1741b66aac
Status: Downloaded newer image for python:3.9-alpine3.13
docker.io/library/python:3.9-alpine3.13
```

So I will use `docker.io/library/python:3.9-alpine3.13@sha256:437482dd314471f5f10967f7376489d9e7f67c111850c123cc725a1741b66aac`